packages/form-js-editor/src/features/properties-panel/groups/ValidationGroup.js

```diff
@@ -274,7 +274,12 @@ function ValidationType(props) {
   const debounce = useService('debounce');
 
   const setValue = (validationType) => {
-    onChange('validationType')(validationType || undefined);
+
+    if (!validationType) {
```

Comment:
From a quick look, this probably doesn't do anything? validationType is only nullish if the value is `VALIDATION_TYPE_OPTIONS.custom.value OR ''` which is the same thing as `validationType` in this context, right? In this case we would still want to use `|| undefined` I think, which will instead of setting the schema to an empty string, entirely clear the property. Though that's optional.

Reply:
You are right, this was a leftover from my root causing. I remove the change to keep it untouched, makes sense at ist.